app/modules/[moduleName]/loading.tsx

```
@@ -11,52 +10,59 @@ import { Edit, MessageSquare, Trash, Upload } from "lucide-react";
-          <h2 className="text-lg font-semibold mb-2">Content</h2>
-          <div className="text-muted-foreground p-4 rounded min-h-[158px] bg-muted">
+          <h2 className="text-lg mb-2">Content</h2>
+          <div className="flex items-center justify-center min-h-[158px] bg-background">
+            <Loader2 className="h-5 w-5 animate-spin" />
           </div>
+          {/* <div className="flex items-center justify-center pt-12">
+            <Loader2 className="h-5 w-5 animate-spin items-center justify-center" />
+          </div> */}
         </div>
 
         <div className="my-6 h-px bg-border"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)